recipes/sdl_image/all/conanfile.py

```diff
@@ -96,17 +96,17 @@ def layout(self):
 
     def requirements(self):
         # Headers are exposed https://github.com/conan-io/conan-center-index/pull/16167#issuecomment-1508347351
-        self.requires("sdl/2.28.3", transitive_headers=True)
+        self.requires("sdl/2.28.5", transitive_headers=True)
         if self.options.with_libtiff:
             self.requires("libtiff/4.6.0")
         if self.options.with_libjpeg:
             self.requires("libjpeg/9e")
         if self.options.with_libpng:
-            self.requires("libpng/1.6.40")
+            self.requires("libpng/1.6.43")
```

Comment:
```suggestion
            self.requires("libpng/[>=1.6 <2]")
```
libpng is already accepting version ranges, this would be better for the future